trieste/acquisition/multiobjective/function.py

```diff
@@ -0,0 +1,25 @@
+from abc import abstractmethod
+
+import tensorflow as tf
+
+from ...type import TensorType
+from ...utils.pareto import Pareto
+from ..function import SingleModelAcquisitionBuilder
+
+
+class HypervolumeAcquisitionBuilder(SingleModelAcquisitionBuilder):
+    @abstractmethod
+    def _calculate_ref_pt(self, pareto: Pareto, ref_pt_calc_method="default"):
+        """
+        calculate the reference point for hypervolme calculation
+        :param pareto: Pareto class
+        :param ref_pt_calc_method
+        """
```

Comment:
did you have a particular use case for this?

Reply:
Was earlier think about this because one may have different acquisition function based on `HypervolumeAcquisitionBuilder`, but maybe for now it is better to remove this part.